emulsify.drush.inc

```diff
@@ -0,0 +1,299 @@
+<?php
+
+/**
+ * @file
+ * Contains Drush hooks. Inspired by Aeon drush commands.
+ */
+
+/**
+ * Implements hook_drush_command().
+ */
+function emulsify_drush_command() {
+  $items = array();
+
+  $items['emulsify'] = array(
+    'description' => 'Create an Emulsify subtheme.',
```

Comment:
I'd say "emulsify based theme." The community has a preconceived notion of "subtheme" and we're not subtheming emulsify